openapi/platform/platform.yaml

```
@@ -4014,9 +4015,10 @@ components:
               description: |-
                 The status of the order will be automatically set to one of the following by the system:
                 - `draft`: The order has not yet been sent for approval
+                - `submitted`: The order is ready for admin approval
```

I feel like the reason that we left out the submitted status originally was we don't know who it should be submitted to. For some partners, it is submitted to the SMB for approval. Others send it to an admin. 

```suggestion
                - `submitted`: The order is ready for approval
```